CONTRIBUTING.md

```
@@ -0,0 +1,76 @@
+```bash
+export WORKSOME_API_TOKEN=...   # a Personal Access Token
+make sync-schema                # introspects the API
+make generate
+make test
+```
+
+`make sync-schema` reads the token from the environment only — deliberately,
+so it never lands in your shell history or the process list. If the API has
+introduced a scalar we don't know about, generation fails and tells you which:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)